docs/developers/CMake.md

```diff
@@ -0,0 +1,37 @@
+# CMake options
+
+<!-- Pure markdown doesn't support code blocks within tables -->
```

Comment:
Perhaps simple bullet-point list would be a better option? Also not sure how html blocks would show up on our website.